api/bazel/repository_locations.bzl

```diff
@@ -100,9 +100,9 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "OpenTelemetry Proto",
         project_desc = "Language Independent Interface Types For OpenTelemetry",
         project_url = "https://github.com/open-telemetry/opentelemetry-proto",
-        version = "0.11.0",
-        sha256 = "985367f8905e91018e636cbf0d83ab3f834b665c4f5899a27d10cae9657710e2",
-        release_date = "2021-10-07",
+        version = "0.14.0",
+        sha256 = "eec079d4170cf4c107046599f79ae6d015fdfcb5fe748126f2f153d73259d700",
+        release_date = "2022-03-08",
```

Comment:
`0.15.0` has just been released - https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v0.15.0

Reply:
Updated this to 0.15.0 👍 

I noticed that the bump to 0.15.0 deprecated a field used in this code (and in the access_logger), `InstrumentationLibrary{Logs,Spans}` in favor of `Scope{Logs,Spans}`, and it's planned to remove the former in June. Should I update those here as well, or should that be a part of the version update when those fields are actually removed? 

Wasn't sure if that would be too many extra changes here (and I'd also be happy to do so in a follow up).

Reply:
Suggest splitting this into a separate PR.